src/UIKit/UIGestureRecognizer.cs

```diff
@@ -76,6 +77,21 @@ public Token ()
 			}
 		}
 
+		[Register ("__UIGestureRecognizerGenericCB")]
+		public class Callback<T> : Token where T: UIGestureRecognizer {
```

Comment:
We probably want this to me internal so that our users do not use it:
```suggestion
		internal class Callback<T> : Token where T: UIGestureRecognizer {
```